site/src/components/pages/hub/BlockDataContainer.tsx

```diff
@@ -58,119 +61,178 @@ export const BlockDataContainer: VoidFunctionComponent<
   }, [text, schema]);
 
   return (
-    <Box
-      mt={5}
-      sx={{
-        display: { xs: "flex", md: "grid" },
-        flexDirection: { xs: "column", md: "unset" },
-        gridTemplateColumns: { md: "60% 40%" },
-      }}
-    >
-      <Box>
-        <Box sx={{ height: 320, backgroundColor: "white" }}>
-          <Tabs
-            disableRipple
-            disableTouchRipple
-            TabIndicatorProps={{
-              style: { display: "none" },
-            }}
-            sx={{
-              "& .MuiTab-root": {
-                textTransform: "none",
-                color: "#64778C",
-              },
-              "& .MuiTab-root.Mui-selected": {
-                backgroundColor: "#F7FAFC",
-                color: "#6048E5",
-                borderTopLeftRadius: 6,
-                borderTopRightRadius: 6,
-              },
-            }}
-            value={blockTab}
-            onChange={(_event, newValue: number) => setBlockTab(newValue)}
-          >
-            <Tab label={metadata.displayName} />
-          </Tabs>
-          <TabPanel value={blockTab} index={0}>
-            <Box
-              display="flex"
-              alignItems="center"
-              sx={(theme) => ({
-                backgroundColor: "#F7FAFC",
-                height: "100%",
-                padding: theme.spacing(0, 4),
-                overflow: "auto",
-              })}
+    <>
+      {isMobile && (
+        <Tabs
+          value={activeMobileTab}
+          onChange={(_event, newValue: number) => setActiveMobileTab(newValue)}
+          sx={{
+            "&.MuiTabs-root": {
+              backgroundColor: theme.palette.gray[10],
+              border: `1px solid ${theme.palette.gray[20]}`,
+              borderRadius: "6px",
+              display: "flex",
+              position: "relative",
+            },
```

Comment:
This is equivalent to:
```suggestion
          sx={{
              backgroundColor: theme.palette.gray[10],
              border: `1px solid ${theme.palette.gray[20]}`,
              borderRadius: "6px",
              display: "flex",
              position: "relative",
```
because this will always have the `MuiTabss-root` CSS class

Reply:
Fixed here acef6d4